src/java/yaw/engine/mesh/Texture.java

```diff
@@ -48,18 +49,34 @@ public void init() {
             try {
                 lInputStream = Texture.class.getResourceAsStream(mFileName);
                 // Load Texture file
+
                 PNGDecoder mDecoder = new PNGDecoder(lInputStream);
 
                 this.mWidth = mDecoder.getWidth();
                 this.mHeight = mDecoder.getHeight();
+                ByteBuffer lByteBuffer;
```

Comment:
source of this modification ?  please indicate where you found this workaround (provide an URL as a comment)